cli/cmd/replicaset.go

```diff
@@ -62,27 +65,263 @@ func NewReplicasetCmd() *cobra.Command {
 		},
 		Args: cobra.ExactArgs(1),
 	}
-	statusCmd.Flags().BoolVar(&orchestratorCentralizedConfig, "config", false,
+
+	addOrchestratorFlags(cmd)
+	addTarantoolConnectFlags(cmd)
+	return cmd
+}
+
+// newPromoteCmd creates a "replicaset promote" command.
+func newPromoteCmd() *cobra.Command {
+	promoteCmd := &cobra.Command{
+		Use: "promote [--cartridge|--config|--custom] [-f] [--timeout secs] [flags] " +
+			"(<APP_NAME:INSTANCE_NAME> | <URI>)\n\n" +
+			"  The URI can be specified in the following formats:\n" +
+			"  * [tcp://][username:password@][host:port]\n" +
+			"  * [unix://][username:password@]socketpath\n" +
+			"  To specify relative path without `unix://` use `./`.",
+		DisableFlagsInUseLine: true,
+		Short:                 "Promote an instance",
+		Long: "Promote an instance.\n\n" +
+			"The command supports the following environment variables:\n\n" +
+			"* " + connect.TarantoolUsernameEnv + " - specifies a username\n" +
+			"* " + connect.TarantoolPasswordEnv + " - specifies a password\n" +
+			"\n",
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdCtx.CommandName = cmd.Name()
+			err := modules.RunCmd(&cmdCtx, cmd.CommandPath(), &modulesInfo,
+				internalReplicasetPromoteModule, args)
+			handleCmdErr(cmd, err)
+		},
+		Args: cobra.ExactArgs(1),
+	}
+
+	addOrchestratorFlags(promoteCmd)
+	addTarantoolConnectFlags(promoteCmd)
+	promoteCmd.Flags().BoolVarP(&replicasetForce, "force", "f", false,
+		"skip instances not found locally")
+	promoteCmd.Flags().IntVarP(&replicasetTimeout, "timeout", "",
+		replicasetcmd.DefaultTimeout, "timeout")
+
+	return promoteCmd
+}
+
+// newConfigSourcePromoteCmd creates a "replicaset config_source promote" command.
+func newConfigSourcePromoteCmd() *cobra.Command {
+	uriHelp := fmt.Sprintf(`The URI specifies a etcd or tarantool config storage `+
+		`connection settings in the following format:
+http(s)://[username:password@]host:port[/prefix][?arguments]
+
+* prefix - a base path to Tarantool configuration in etcd or tarantool config storage.
+
+Possible arguments:
+
+* key - a target configuration key in the prefix.
+* name - a name of an instance in the cluster configuration.
+* timeout - a request timeout in seconds (default %.1f).
+* ssl_key_file - a path to a private SSL key file.
+* ssl_cert_file - a path to an SSL certificate file.
+* ssl_ca_file - a path to a trusted certificate authorities (CA) file.
+* ssl_ca_path - a path to a trusted certificate authorities (CA) directory.
+* ssl_ciphers - a colon-separated (:) list of SSL cipher suites the connection can use.
+* verify_host - set off (default true) verification of the certificate’s name against the host.
+* verify_peer - set off (default true) verification of the peer’s SSL certificate.
+
+You could also specify etcd username and password with environment variables:
+* %s - specifies an etcd username
+* %s - specidies an etcd password
```

Comment:
We need also add variables for tarancool config storage to the help.

Reply:
Added.